### PR TITLE
Remove "beta" references to app components documentation.

### PR DESCRIPTION
**Pull request by aw-asana**

### CODE REVIEW

**Review by jv-asana — APPROVED** [review]
LGTM. I guess we could remove the `beta-indicator` css if we aren't planning on using it but could also just be worth keeping it around just in case.

**Comment by aw-asana** [comment]
Renamed it to `preview-indicator` because that's how we actually use it now. @jv-asana

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -109,7 +109,7 @@ under the License.
               <% if api_reference %><a class="toc-h1 toc-link"><span class="sidebar-section">REST API</span></a><% end %>
               <% if platform_ui_alpha %><a class="toc-h1 toc-link"><span class="sidebar-section">App Components</span></a></li><li class="platform-ui-alpha"><% end %>
```
Comment (jv-asana) [inline source/layouts/layout.erb:110]: Should we also do something about this `platform_ui_alpha` now that we removed the beta?

Comment (aw-asana) [reply source/layouts/layout.erb]: Fixed!

**Review by jv-asana — APPROVED** [review]
LGTM

**Pull request merged**